src/Components/GraphiQLField/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+import { Grid, Button, ButtonGroup } from "@mui/material";
+import RequestField from "../RequestField";
+import { DEF_VALUE_REQUEST } from "./constants";
+import { ErrorFallbackComponent } from "../ErrorFallbackComponent";
+import { ErrorBoundary } from "react-error-boundary";
+import ResponseFieldWithApollo from "../ResponseFieldWithApollo";
+import { PlayArrow } from "@mui/icons-material";
+import Schema from "../Schema";
+import { ErrorModalWindow } from "../ErrorModalWindow";
+import { useTranslation } from "react-i18next";
+
+interface IRequest {
```

Comment:
redundant `I` prefix
```suggestion
interface Request {
```